projects/packages/connection/src/class-users-connection-admin.php

```diff
@@ -112,7 +113,7 @@ public function enqueue_scripts( $hook ) {
 			'jetpack-users-connection',
 			'jetpackConnectionTooltips',
 			array(
-				'columnTooltip' => esc_html__( 'Connecting a WordPress.com account unlocks Jetpack’s full suite of features including secure logins.', 'jetpack-connection' ),
+				'columnTooltip' => esc_html__( 'Connecting a WordPress.com account unlocks Jetpack\'s full suite of features including secure logins.', 'jetpack-connection' ),
```

Comment:
How about a real apostrophe instead?

```suggestion
				'columnTooltip' => esc_html__( 'Connecting a WordPress.com account unlocks Jetpack’s full suite of features including secure logins.', 'jetpack-connection' ),
```

Reply:
Yeah, that works 👍